### DISCUSSION TITLE
How to add module blade to the PowerEdge M1000e

### DISCUSSION THREAD

**Discussion by raad-orfali** [netbox-community/devicetype-library#1881] [Q&A]
I couldn't find any type of modules for the Dell M1000e component neither blades nor back switches and how I can add them to the Netbox

**Comment by danner26** [comment]
The device is here: [Dell PowerEdge-M1000e.yaml](https://github.com/netbox-community/devicetype-library/blob/704d4c60c16bcb5fef1ffda7c06c97833ebb050a/device-types/Dell/PowerEdge-M1000e.yaml#L21)

If there are no sub-devices you are welcome to model them in the yaml format and submit a PR which we can review
